Source/JavaScriptCore/ftl/FTLLowerDFGToB3.cpp

```diff
@@ -13207,8 +13207,6 @@ IGNORE_CLANG_WARNINGS_END
                 RefPtr<OSRExitHandle> handle = descriptor->emitOSRExitLater(
                     *state, UncountableInvalidation, origin, params, nodeIndex, 0);
 
-                RefPtr<JITCode> jitCode = state->jitCode.get();
```

Comment:
The link task below seems to use state without issue, so it seems to me like state should keep it alive. Truthfully I don't know why this is even building today, because the variable is unused.

Reply:
```// None of these things is owned by State. It is the responsibility of
    // FTL phases to properly manage the lifecycle of the module and function.```

Reply:
I guess I will explicitly use it instead